Examples/minimum_user_for_s2e/CMakeLists.txt

```
@@ -167,18 +86,17 @@ else()
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/src/src_user/Script)
 endif()
 
+include(../../common.cmake)
```

[MUST] これだと，現行のS2E on VSだとビルドできなさそう．

というのは，現在だと，
```
./S2E/FlightSW/c2a_sample/
```
に， userのsrcフォルダの１つ上がマウントされるからです．

なので，

```suggestion
include(./src/src_core/common.cmake)
```
 とか？

---

あ，これはミスですね

---

ただ，coreとuserで`CMakeLists.txt`分けたこと自体がそもそも`src/{src_core|src_user}`をなんとかしたい(#6 )，というところから来てるのもあるので，`C2A_CORE_DIR`みたいな変数作ってそこから参照しようかなあ

---

`C2A_CORE_DIR`と`C2A_USER_DIR`を作り，`common.cmake`をcoreとuserで分割しました．
各ソースファイルは`${C2A_USER_DIR}/common.cmake`を見に行き，`common.cmake`は`${C2A_CORE_DIR}/common.cmake`を見に行きます．